website/docs/exhaustiveness.md

```diff
@@ -3,15 +3,12 @@ id: exhaustiveness
 title: Exhaustiveness Checking
 ---
 
-> **Note**: Sorbet does not currently have first-class support for
-> exhaustiveness checks. This feature is planned, but has not yet been
-> implemented. Here we document a current workaround.
+**Exhaustiveness checking** is a feature of a language where the type checker
```

Comment:
WDYT about separating docs update and landing them when we verified the feature on a few codebases?